exe/main.cc

```diff
@@ -26,7 +23,17 @@ namespace HW = HorseWhisperer;
 namespace lth_file = leatherman::file_util;
 
 int startAgent(std::vector<std::string> arguments) {
-#ifndef _WIN32
+#ifdef _WIN32
+    try {
+        Util::daemonize();
```

Comment:
Foreground should allow multiple instances to run?

On Windows it's not going to start a process in the background, as that's not really how services work.